gc/scheduler/scheduler.go

```diff
@@ -255,7 +255,6 @@ func (s *gcScheduler) run(ctx context.Context) {
 		interval    = time.Second
 		gcTime      time.Duration
 		collections int
-		// TODO(dmcg): expose collection stats as metrics
```

Comment:
I think we may need more stats such as `stats.Elapsed()` before removing the comment. (cc @dmcgowan)